frontend/sozisel-app/src/components/SessionResultScreen/SessionResultEvents/EventResultDetails/Modules/QuizResultDetails/QuizResultViews/Questions/QuizResultQuestionsView.tsx

```diff
@@ -0,0 +1,148 @@
+import "./QuizResultQuestionView.scss";
+
+import { CircularProgress, Typography } from "@material-ui/core";
+import {
+  QuizQuestionSummary,
+  useQuizQuestionsSummaryQuery,
+} from "../../../../../../../../graphql";
+import React, { useState } from "react";
+
+import AssignmentIcon from "@material-ui/icons/Assignment";
+import EnhancedTable from "../../../../../../../utils/Table/Table";
+import ErrorAlert from "../../../../../../../utils/Alerts/ErrorAlert";
+import QuizResultDetailsDialog from "../DialogUtils/QuizResultDetailsDialog";
+import { ensure } from "../../../../../../../utils/Typescript/ensure";
+import { useTranslation } from "react-i18next";
+
+export interface QuizResultQuestionViewProps {
+  id: string;
+}
+export default function QuizResultQuestionView({
+  id,
+}: QuizResultQuestionViewProps): React.ReactElement {
+  const { t } = useTranslation("common");
+  const { loading, data } = useQuizQuestionsSummaryQuery({ variables: { id } });
+  const [currentQuestion, setCurrentQuestion] =
+    useState<QuizQuestionSummary | null>(null);
+
+  const getQuestionStats = () => {
```

Comment:
wtf x2?

Reply:
I would understand if you place the same variable even without memo as creating it is fast, but what have come to your head to create a function for that? To much java and getters or what?